ClinGen/clincoded
# 2124 Change case segregation PHI

### Updates

Since howard's PR is merged, this PR synced with that and now real js file changes decrease down to just one.

---

### Original Post

Notes:
- Part of this pull request borrows changes from #2153 (issue #2108), you may want to review this PR after #2153 is merged into dev and this PR sync with dev.
- This pull request should be merged after #2153  is merged.
- Database dump is required by term of use page, kudos to Howard's work from #2108: https://github.com/ClinGen/clincoded/compare/2108_ht_terms_of_use_page

Test steps for curator testing:
1. Go into VCI, choose a variant and go to the detail page.
2. Create an interpretation.
3. Switch to the "Case Segregation" tab, you should now see the yellow warning panel for the PHI disclaimer.
4. Click on the "term of use" link, and the link should link you to the term of use page in a new tab.
5. Click on the link "protected health information (PHI)" should bring you to the PHI website in a new tab.
6. Both link you should see an external link icon.
